nixos/modules/programs/corefreq.nix

```diff
@@ -0,0 +1,42 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.corefreq;
+  kernelPackages = config.boot.kernelPackages;
```

Comment:
Any reason for this variable? It seems like this is only used in the package option.